language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/LSPackageLoader.java

```diff
@@ -150,28 +151,31 @@ private List<PackageInfo> getPackagesFromRepository(PackageRepository repository
         Map<String, List<String>> packageMap = repository.getPackages();
         List<PackageInfo> packages = new ArrayList<>();
         packageMap.forEach((key, value) -> {
-            if (key.equals(Names.BALLERINA_INTERNAL_ORG.getValue())) {
-                return;
-            }
-            value.forEach(nameEntry -> {
-                String[] components = nameEntry.split(":");
-                if (components.length != 2 || skipList.contains(components[0])) {
+            try {
+                if (key.equals(Names.BALLERINA_INTERNAL_ORG.getValue())) {
                     return;
                 }
-                String nameComponent = components[0];
-                String version = components[1];
-                PackageOrg packageOrg = PackageOrg.from(key);
-                PackageName packageName = PackageName.from(nameComponent);
-                PackageVersion pkgVersion = PackageVersion.from(version);
-                PackageDescriptor pkdDesc = PackageDescriptor.from(packageOrg, packageName, pkgVersion);
-                ResolutionRequest request = ResolutionRequest.from(pkdDesc, PackageDependencyScope.DEFAULT);
-
-                Optional<Package> repoPackage = repository.getPackage(request,
-                        ResolutionOptions.builder().setOffline(true).build());
-                repoPackage.ifPresent(pkg -> packages.add(new PackageInfo(pkg)));
-            });
+                value.forEach(nameEntry -> {
+                    String[] components = nameEntry.split(":");
+                    if (components.length != 2 || skipList.contains(components[0])) {
+                        return;
+                    }
+                    String nameComponent = components[0];
+                    String version = components[1];
+                    PackageOrg packageOrg = PackageOrg.from(key);
+                    PackageName packageName = PackageName.from(nameComponent);
+                    PackageVersion pkgVersion = PackageVersion.from(version);
+                    PackageDescriptor pkdDesc = PackageDescriptor.from(packageOrg, packageName, pkgVersion);
+                    ResolutionRequest request = ResolutionRequest.from(pkdDesc, PackageDependencyScope.DEFAULT);
+
+                    Optional<Package> repoPackage = repository.getPackage(request,
```

Comment:
What if we wrap this part? Since this is the part that's going to fail? That way, only the erroneous packages won't be loaded. (Ignore my last comment about try if we wrap this part only)